src/runtime/auto-prompt-manager.ts

```diff
@@ -997,7 +1023,16 @@ export class AutoPromptManager {
    */
   private async getActionImpressions_(actionType: string): Promise<number[]> {
     if (!ACTION_TO_IMPRESSION_STORAGE_KEY_MAP.has(actionType)) {
-      // TODO: handle missing storage key for action
+      this.eventManager_.logEvent({
+        eventType:
+          AnalyticsEvent.EVENT_ACTION_IMPRESSIONS_STORAGE_KEY_NOT_FOUND_ERROR,
+        eventOriginator: EventOriginator.SWG_CLIENT,
+        isFromUserAction: false,
+        additionalParameters: {
+          publicationid: this.pageConfig_.getPublicationId(),
+          actionType,
```

Comment:
I don't think this support arbitrary parameters like this. Do they need to be registered in the proto: http://google3/logs/proto/subscribewithgoogle/client/subscribewithgoogle_client_extension.proto;l=602;rcl=568642159?

Reply:
This was my initial impression, but I saw some other usages of `additionalParameters` so I got optimistic... I will remove these parameters.